### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by finnp** [Fankserver/atom-firepad#11]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:54:11)
FirepadView.initialize (/Users/finn/.atom/packages/firepad/lib/firepad.coffee:28:9)
```

**Comment by Fank** [comment]
This should be fixed in 0.4.0

**Comment by finnp** [comment]
:+1:
